### ISSUE TITLE
V-Day: NSFW product images

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#2518]
There are concerns that some of the new product images might 'surprise' some of our viewers. We want to be sex positive and don't want to hid away from it too much but at the same time respect users who just don't want to see that.

We're thinking of blurring some of the images and possibly adding a NSFW badge to them. But ideally, we would like to show the modified blurred image for thumbnails (homepage and category pages) but then show the real image on the product page.

How big of a lift would it be to add an option in the CMS for an optional field to upload a different thumbnail image?

If you are curious, you can see the products at the bottom of the page https://foundation.mofostaging.net/en/privacynotincluded/categories/Smart%20Home/

cc @mmmavis @Pomax @alanmoo

**Comment by kristinashu** [comment]
Good point! @beccaklam @natalieworth are exploring different image treatments now. I guess we'll need to see what they come up with. If it's just a simple blur and a NSFW badge then just a check box in the CMS should work. But if it's a more complicated image treatment (i.e. not achievable through CSS or unique for each product) then we'd need two separate images.

**Comment by Pomax** [comment]
Note that the filter function [isn't very widely supported](https://caniuse.com/#search=filter()), though, so we might need "real" blurs as fallback anyway.

**Comment by mmmavis** [comment]
Ummm this was the one I checked: https://caniuse.com/#feat=css-filters seems fine to me 👀

**Comment by kristinashu** [comment]
We should have design for this done today, then we can decide what how to proceed.

**Comment by natalieworth** [comment]
Heart icon is here:
[18-plus-heart-icon.svg.zip](https://github.com/mozilla/foundation.mozilla.org/files/2809425/18-plus-heart-icon.svg.zip)

Mockups are in [redpen here](https://redpen.io/em8bd3dbf9aa17490d)
<img width="338" alt="screen shot 2019-01-29 at 11 54 46 am" src="https://user-images.githubusercontent.com/14876567/51936295-b6404580-23bc-11e9-9b93-507425166087.png">

I'm not 100% sure how the gaussian blur works—but I think it's set by pixel amount? @mmmavis Maybe try 5px. I think sometimes blur affects also cause the background to feather into white. If that happens, is it possible to set the background color to #f4f4f4 on the 18+ images so it doesn't have a white outline around the edges? White edge issue kind of looks like this:
<img width="836" alt="screen shot 2019-01-29 at 12 05 16 pm" src="https://user-images.githubusercontent.com/14876567/51936955-34e9b280-23be-11e9-8ee6-e4262514a92c.png">

**Comment by mmmavis** [comment]
@natalieworth let's play around with the CSS config together in the office today!

**Comment by mmmavis** [comment]
@natalieworth product image itself is always on transparent background so I _think_ it shouldn't have that white edge issue.

**Comment by kristinashu** [comment]
Images don't have transparent background. They're on a gray. If you want to test them, all the new images are [here](https://drive.google.com/drive/folders/1heDtdi5UmHUqA31Gp8pK4uvVwImBOT-F?usp=sharing).
